tests/testthat/test-batch_downloader.R

```diff
@@ -0,0 +1,157 @@
+test_that("batch_downloader works as expected", {
+  skip("batch_downloader requires API call")
+
+  api_call_n <- paste0(url, "api/outbreaks/", outbreak_id, "/cases/count")
+  api_call_get <- paste0(url, "api/outbreaks/", outbreak_id, "/cases")
+  res <- batch_downloader(
+    url = url,
+    username = username,
+    password = password,
+    api_call_n = api_call_n,
+    api_call_get = api_call_get,
+    batch_size = 50000
+  )
```

Comment:
This test fails for me: `Error in while (skip < num_records) { : argument is of length zero` - when I have attempted to set the outbreak id to the one given in the examples, using `set_outbreak_id()`. It works when the outbreak id is set using `get_active_outbreak()` - unsure why this is happening.